tests/Fixtures/Factories/RichPostFactory.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Zenstruck\Foundry\Tests\Fixtures\Factories;
+
+use Zenstruck\Foundry\Instantiator;
+use Zenstruck\Foundry\ModelFactory;
+use Zenstruck\Foundry\Tests\Fixtures\Entity\RichPost;
+
+final class RichPostFactory extends ModelFactory
+{
+    protected function initialize()
+    {
+        return $this->instantiateWith((new Instantiator())->alwaysForceProperties());
```

Comment:
nice, I didn't know this one. This way no need for any setter :tada:

Reply:
You can set this globally in your config too.